src/platform/user/profile/vet360/containers/AddressValidationModal.jsx

```diff
@@ -69,8 +66,13 @@ class AddressValidationModal extends React.Component {
       validationKey,
       isLoading,
       addressFromUser,
+      confirmedSuggestions,
+      selectedAddressId,
     } = this.props;
 
+    const disableButton =
```

Comment:
It feels like we should simply be checking to see if the `selectedAddressId` is set to anything at all

Reply:
We can't go that route because it defaults to something in order to make the first suggested address selected.  We could change up the logic so it defaults to null and we set it if there is suggestedAddresses